apps/transport/lib/validators/gtfs_transport_validator.ex

```diff
@@ -124,9 +132,16 @@ defmodule Transport.Validators.GTFSTransport do
 
   def count_by_severity(_), do: %{}
 
-  @spec count_by_severity(map()) :: {binary(), integer()}
+  @doc """
+  Returns the maximum severity, with the issues count
+
+  iex> validation_result = %{"tooClose" => [%{"severity" => "Warning"}], "funnyName" => [%{"severity" => "Information"}, %{"severity" => "Information"}], "NullDuration" => [%{"severity" => "Warning"}]}
+  iex> count_max_severity(validation_result)
+  {"Warning", 2}
```

Comment:
```suggestion
  iex> count_max_severity(validation_result)
  {"Warning", 2}
  iex> count_max_severity(%{})
  {"NoError", 0}
```
Ajoute un test (suggéré dans https://github.com/etalab/transport-site/pull/2464/files#r909293782), je crois que tu l'avais ajouté dans `count_by_severity`

Reply:
Peut-être pas nécessaire comme tu as déjà `get_max_severity_error(%{})`, à ta guise